fundamentals/android/auth-sign-in-with-google/Notes/app/src/main/java/com/notes/app/screens/authentication/sign_in/SignInScreen.kt

```diff
@@ -131,32 +137,38 @@ fun SignInScreen(
 
         Spacer(modifier = Modifier
             .fillMaxWidth()
-            .padding(4.dp))
+            .padding(8.dp))
 
         Text(text = stringResource(R.string.or), fontSize = 16.sp, color = Purple40)
 
         Spacer(modifier = Modifier
             .fillMaxWidth()
-            .padding(4.dp))
+            .padding(8.dp))
 
-        AuthenticationButton(R.string.sign_in_with_google) { credential ->
+        AuthenticationButton(buttonText = R.string.sign_in_with_google) { credential ->
             viewModel.onSignInWithGoogle(credential, openAndPopUp)
         }
 
         Spacer(modifier = Modifier
             .fillMaxWidth()
             .padding(8.dp))
 
-        TextButton(onClick = { viewModel.onSignUpClick(openAndPopUp) }) {
+        TextButton(onClick = { viewModel.onSignUpClick(openScreen) }) {
             Text(text = stringResource(R.string.sign_up_description), fontSize = 16.sp, color = Purple40)
         }
+
+        LaunchedEffect(Unit) {
+            launchCredManBottomSheet(context) { result ->
+                viewModel.onSignInWithGoogle(result, openAndPopUp)
+            }
+        }
```

Comment:
nit: can we move this to the top/beginning of this composable function? For better discoverability.

Reply:
Done